### PR TITLE
Release/4.8.0

### PR DESCRIPTION
**Pull request by taquitos**
### New API

* `EntitlementInfo`: added `isActiveInCurrentEnvironment` and `isActiveInAnyEnvironment` (#1755) via NachoSoto (@NachoSoto)

### Other Changes
* Plumb platformInfo in Configuration for PHC use (#1757) via Joshua Liebowitz (@taquitos)
* added a log when `autoSyncPurchases` is disabled (#1749) via aboedo (@aboedo)
* Re-fetch cached offerings and products after Storefront changes (3/4)  (#1743) via Juanpe Catalán (@Juanpe)
* `bug_report.md`: clarify SK2 support (#1752) via NachoSoto (@NachoSoto)
* `logErrorIfNeeded`: also log message if present (#1754) via NachoSoto (@NachoSoto)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 4.8.0
+* Plumb platformInfo in Configuration for PHC use (#1757) via Joshua Liebowitz (@taquitos)
+* `EntitlementInfo`: added `isActiveInCurrentEnvironment` and `isActiveInAnyEnvironment` (#1755) via NachoSoto (@NachoSoto)
+* `logErrorIfNeeded`: also log message if present (#1754) via NachoSoto (@NachoSoto)
+* `bug_report.md`: clarify SK2 support (#1752) via NachoSoto (@NachoSoto)
+* Re-fetch cached offerings and products after Storefront changes (3/4)  (#1743) via Juanpe Catalán (@Juanpe)
+* Prepare next version: 4.8.0-SNAPSHOT (#1750) via RevenueCat Releases (@revenuecat-ops)
+* Fix replace version without prerelease modifiers (#1751) via Toni Rico (@tonidero)
+* added a log when `autoSyncPurchases` is disabled (#1749) via aboedo (@aboedo)
```
Comment (aboedo) [inline CHANGELOG.md:9]: Can we split this into "new APIs", "bug fixes" and "other"? 
We need to automate the process, but for now, it'd be nice to keep the changelogs nice

**Review by aboedo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (taquitos) [reply CHANGELOG.md:9]: <YouGotItDude.gif>

File: CHANGELOG.latest.md [hunk 2]
```diff
@@ -1,28 +1,8 @@
...
-* `Purchases.customerInfo(fetchPolicy:)`: actually use `fetchPolicy` parameter (#1721) via NachoSoto (@NachoSoto)
-* `PurchasesOrchestrator`: fix behavior dealing with `nil` `SKPaymentTransaction.productIdentifier` during purchase (#1680) via NachoSoto (@NachoSoto)
-* `PurchasesOrchestrator.handlePurchasedTransaction`: always refresh receipt data (#1703) via NachoSoto (@NachoSoto)
\ No newline at end of file
+* Plumb platformInfo in Configuration for PHC use (#1757) via Joshua Liebowitz (@taquitos)
+* `EntitlementInfo`: added `isActiveInCurrentEnvironment` and `isActiveInAnyEnvironment` (#1755) via NachoSoto (@NachoSoto)
+* `logErrorIfNeeded`: also log message if present (#1754) via NachoSoto (@NachoSoto)
+* `bug_report.md`: clarify SK2 support (#1752) via NachoSoto (@NachoSoto)
+* Re-fetch cached offerings and products after Storefront changes (3/4)  (#1743) via Juanpe Catalán (@Juanpe)
+* Prepare next version: 4.8.0-SNAPSHOT (#1750) via RevenueCat Releases (@revenuecat-ops)
```
Comment (NachoSoto) [inline CHANGELOG.latest.md:6]: I would remove this one

File: CHANGELOG.md (hunk 1 above)
Comment (taquitos) [reply CHANGELOG.md]: ✅

File: CHANGELOG.latest.md (hunk 2 above)
Comment (taquitos) [reply CHANGELOG.latest.md]: Removed 😄

**Pull request merged**